prow/pluginhelp/pluginhelp.go

```diff
@@ -41,6 +43,14 @@ type PluginHelp struct {
 	// Events is a slice containing the events that are handled by the plugin.
 	// NOTE: Plugins do not need to populate this. Hook populates it on their behalf.
 	Events []string
+	// Commands maps a command name to a struct of its properties.
+	Commands map[string]struct {
```

Comment:
Also unless there's a good reason to use an anonymous `struct` here let's give it a name?

Reply:
yes, I will update it.